src/main/java/janggi/dao/BoardPieceDao.java

```diff
@@ -0,0 +1,127 @@
+        preparedStatement.setString(5, piece.getTeam().name());
+        preparedStatement.executeUpdate();
+    }
+
+    public Board loadBoard(int gameId) {
+        String query = "select * from board_piece where game_id = ?;";
+        try (Connection connection = getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+        ) {
+            preparedStatement.setInt(1, gameId);
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                Map<Position, Piece> board = new HashMap<>();
+                while (resultSet.next()) {
+                    int column = resultSet.getInt("column_value");
+                    int row = resultSet.getInt("row_value");
+                    PieceType pieceType = PieceType.valueOf(resultSet.getString("piece_type"));
+                    Team team = Enum.valueOf(Team.class, resultSet.getString("team"));
+                    Position position = new Position(Column.from(column), Row.from(row));
+
+                    Piece piece = pieceType.toPiece(team);
+                    board.putIfAbsent(position, piece);
+                }
+                return new Board(board);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("DB 오류 발생");
+        }
+    }
+
+    public void delete(int gameId, Position goalPosition) {
+        String query = "delete from board_piece where game_id = ? and column_value = ? and row_value = ?;";
+        try (Connection connection = getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(query)
+        ) {
+            Column column = goalPosition.getColumn();
+            Row row = goalPosition.getRow();
+            preparedStatement.setInt(1, gameId);
+            preparedStatement.setInt(2, column.getValue());
+            preparedStatement.setInt(3, row.getValue());
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("DB 오류 발생");
+        }
+    }
+
+    public void updatePiecePosition(int gameId, Position startPosition, Position goalPosition) {
+        String query = "update board_piece set column_value = ?, row_value = ? where game_id = ? and column_value = ? and row_value = ?;";
+        try (Connection connection = getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(query)
+        ) {
+            Column startColumn = startPosition.getColumn();
+            Row startRow = startPosition.getRow();
+            Column goalColumn = goalPosition.getColumn();
+            Row goalRow = goalPosition.getRow();
+            preparedStatement.setInt(1, goalColumn.getValue());
+            preparedStatement.setInt(2, goalRow.getValue());
+            preparedStatement.setInt(3, gameId);
+            preparedStatement.setInt(4, startColumn.getValue());
+            preparedStatement.setInt(5, startRow.getValue());
```

Comment:
```suggestion
            preparedStatement.setInt(1, goalPosition.getColumn().getValue());
            preparedStatement.setInt(2, n.getRow().getValue());
            preparedStatement.setInt(3, gameId);
            preparedStatement.setInt(4, startPosition.getColumn().getValue());
            preparedStatement.setInt(5, startPosition.getRow().getValue());
```
변수를 따로 둘 필요는 없어보여요. 오히려 행이 많아져서 읽기 어려웠어요.